docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-organizations-hierarchy.md

```diff
@@ -0,0 +1,155 @@
+
+## Is the Organization Hierarchy Mandatory?
+
+No. If no organization hierarchy is configured, all repositories will default to the `All` organization. This setup can be suitable when managing a relatively small number of repositories. However, as your repository count grows, we recommend establishing an organization hierarchy to improve structure and scalability.
+
+## How is the Organization Hierarchy Defined?
+
+Organizational structure is a configured via a `repos.csv` file, accessible to the Agent via the filesystem or network. Depending on your deployment configuration, this may by done by placing the file in the same host as the agent or by using a file/network mount. The Agent can also be configured with an unauthenticated http/s URI that serves the aforementioned `repos.csv`.
+
+This configuration is best explained with an example. Consider an organization that consists of two teams and two Directors:  
+
+```
+All
+├── Director A
+|       └── Team 1
+|       └── Team 2
+├── Director B
+
+```
+
+```csv showLineNumbers
+cloneUrl,branch,org1,org2,org3
+"https://github.com/apache/maven-doxia","master","Team 1","Director A","ALL"
+"https://github.com/Netflix/photon","main","Team 2","Director A","ALL"
+"https://github.com/Netflix/ribbon","master","Director A","ALL"
+"https://github.com/apache/maven-doxia","master","Director B","ALL"
+```
+
+In this example `repos.csv`, we're defining 5 organizations, associating 3 repos with them. Every repository is automatically associated with its parent organizations. In our example, `Director A` owns all 3 repositories. 
+
+:::tip
+
+Comma-separated files (csv) are best manipulated using a dedicated editor such as Microsoft Excel, Google Sheets, or Apple Numbers. Here is what the above csv would look like on Apple Numbers:
+
+![](../assets/numbers-csv-screenshot.png)
+
+:::
+    
+## Expected format for `repos.csv` 
+
+- Use quotation marks around each value to help differentiate between the different columns.
+- A single row may not have more columns than what is defined in the first row.
+- A single repository may be associated with multiple organizations by being referenced in multiple rows. In the above example, `maven-doxia` is associated with both `Director B` and `Team 1`.
+- Repositories can be associated with any organization down the tree, not just the leaves. See line 4 in the `csv` file for an example.
+
+
+:::warning
+Org columns must start immediately after the branch column. 
+
+```csv {showLineNumbers}
+# ❌ Don't do this
+cloneUrl,branch,org1,org2,org3
+"https://github.com/apache/maven-doxia","master","Org 1", "Org2", "Org3"
+"https://github.com/apache/maven","master","","Org2","Org3"
+
+# ✅ This is OK
+cloneUrl,branch,org1,org2,org3
+"https://github.com/apache/maven-doxia","master","Org 1", "Org2", "Org3"
+"https://github.com/apache/maven","master","Org2","Org3"
```

Comment:
I found this a little confusing. I think the intention was showing that you shouldn't have a "" after master -- but who would do that? Like what would the intention of a user be? 

Also, I find it a bit strange we have spaces between the orgs. Why do we have spaces there but not elsewhere?

Similar question with the warning below here.